### PR TITLE
Pr merge script

### PR DESCRIPTION
**Pull request by mvacanti**
Reference to  Auterion https://github.com/Auterion/px4-jsbsim-bridge/pull/18

This proposal is a combination of the work from #7Re-Architect JSBSim Scenario Loading and latest thoughts:

- Does not change default loading of aircraft and world behavior
- If user includes `<jsb_script>[user script.xml]</jsb_script>` in configs file the script will load
- Does not resolve current JSBSim limitation of loading aircraft from script with path and file name are not the same i.e Rasal/Rascal110JSBSim.xml but does alert user to this condition.

### CODE REVIEW

File: configs/quadrotor_x.xml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <model name="quadrotor_x">
+    <jsbsimbridge>
+        <jsb_script>motor_0_fail.xml</jsb_script>
+    </jsbsimbridge>
```
Comment (Jaeyoung-Lim) [inline configs/quadrotor_x.xml:4]: Does this mean that the motor will always fail for the quadrotor_x model?

Comment (mvacanti) [reply configs/quadrotor_x.xml:4]: Yes, in this case I included that change for the purpose of presenting a working solution. A final version would not include. The intent would be that the default behavior for all airframes is without the script.

**Review by Jaeyoung-Lim — APPROVED** [review]
Thanks!

This is really awesome that we can now have failure scenarios handled from the simulation.

**Comment by Jaeyoung-Lim** [comment]
@mvacanti I tried to resolve the conflicts with https://github.com/Auterion/px4-jsbsim-bridge/pull/19 myself, but it broke the quadrotor_x when running with latest PX4/master with
```
make px4_sitl jsbsim_quadrotor_x
```

Would you mind probably rebasing the PR? thanks

**Comment by mvacanti** [comment]
@Jaeyoung-Lim rebased and made a few minor improvements. Try again?

**Comment by Jaeyoung-Lim** [comment]
@mvacanti Thanks!

**Pull request merged**